adcm_aio_client/core/config/_base.py

```diff
@@ -0,0 +1,346 @@
+from abc import abstractmethod
```

Comment:
Do we plan to add License?

Reply:
Yes, as I know, but it should be done in one place with licence checkring, etc.